dracut/30ignition/ignition-files.service

```diff
@@ -5,30 +5,14 @@ DefaultDependencies=false
 # Ignition files stage starts after partitioning is completed.
 Requires=initrd-root-fs.target
 After=initrd-root-fs.target
+Before=initrd-parse-etc.service
+
+Requires=ignition-disks.service
+After=ignition-disks.service
```

Comment:
This should be already implicit in the `After=initrd-root-fs.target` above. We can leave this explicit for the sake of clarity if we want. Maybe we can merge these with the lines above though, as the comment there already explains it.